hail/src/main/scala/is/hail/expr/types/physical/PType.scala

```diff
@@ -25,7 +25,7 @@ object PType {
 
   val genRequiredScalar: Gen[PType] = genScalar(true)
 
-  def genComplexType(required: Boolean): Gen[ComplexPType] = {
+  def genComplexType(required: Boolean): Gen[PType] = {
```

Comment:
My first attempt at removing ComplexPType from PCall,Locus,Interval was to simply to mix it in with the canonical implementations of those interfaces. This mostly worked, except here, where a PCall and PLocus were added to the same Seq.

To fix this, a simple PType here suffices. However, this function doesn't appear to be checked, because genComplexType is only used for genInsertable, which is not used anywhere (and not tested).

I can remove these if they're not needed.

![Screenshot 2019-12-16 11 53 52](https://user-images.githubusercontent.com/5543229/70926287-d68be980-1ffa-11ea-8891-400387ffdfdf.png)
![Screenshot 2019-12-16 11 53 58](https://user-images.githubusercontent.com/5543229/70926288-d68be980-1ffa-11ea-8eb5-2a8a0bd88622.png)